src/main/java/com/monkeyrizz/codeforest/controller/ProblemController.java

```diff
@@ -20,8 +21,16 @@ public ProblemController(ProblemService problemService) {
     }
 
     @GetMapping
-    @RequestMapping(path = "")
+    @RequestMapping(path = "list")

```

Comment:
`list` doesn't seem like a great name for the request mapping -- can we update this to be something more intentional?